### PR TITLE
Update Parse

### PR DESCRIPTION
**Pull request by swarthout**
Parse now includes Realtime from Server and has React Native Support
links:
Live Queries: http://blog.parse.com/announcements/parse-server-goes-realtime-with-live-queries/

React Native: https://github.com/ParsePlatform/Parse-SDK-JS/blob/master/README.md 
(read "Using Parse on Different Platforms")

### CODE REVIEW

**Comment by katopz** [comment]
also dashboard? http://blog.parse.com/announcements/introducing-the-parse-server-dashboard/

**Comment by swarthout** [comment]
@katopz Also true. I will add that to the PR

**Comment by daffl** [comment]
Thank you! I like that those comparisons keep us up to date with what's going on :smile:
We probably want to update the written comparison at https://github.com/feathersjs/feathers-docs/blob/master/why/vs/parse.md, too.

File: why/vs/readme.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ If you find something invalid or out of date in the comparisons, please [create
             <td>❌ 🤓</td>
             <td>❌</td>
             <td>❌</td>
-            <td>❌</td>
+            <td>✅</td>
```
Comment (ekryski) [inline why/vs/readme.md]: mmm sort of. You can get things to get notified about things over sockets but you can't send data over sockets. I feel like it's more of half check.

Thanks for the PR though.

Comment (ekryski) [inline why/vs/readme.md]: My bad. I'm an idiot. Was looking at the wrong column!

**Pull request merged**